src/utils/defaultCreator.ts

```diff
@@ -0,0 +1,8 @@
+import { Creator } from '../types/types';
+
+export const defaultCreator: Creator = {
+  id: 0,
+  name: 'NonName',
+  username: 'noUsername',
+  email: 'noEmail@gmail.com',
+};
```

Comment:
Redundant

Reply:
I made it only for ts, because he think that .find() can return undefined